Sources/FoundationEssentials/Locale/Locale+Components.swift

```diff
@@ -508,8 +508,6 @@ extension Locale {
             _normalizedIdentifier = identifier.lowercased()
         }
 
-        package static let latn = NumberingSystem("latn")
```

Comment:
Since this is not a public declaration, I decided to rename it and [declare](https://github.com/swiftlang/swift-foundation/pull/1055/files#diff-e2f80bda044eb123351d364a67c164d3f52c1501163f0e4a9539e34c098d68cfR2177) it along with other variables